modules/ROOT/pages/deploy-on-premises.adoc

```diff
@@ -125,7 +125,11 @@ mvn clean package deploy -DmuleDeploy
 
 include::mule-runtime::partial$mmp-concept.adoc[tag=authenticationMethodsIntro]
 
+<<<<<<< HEAD
 include::mule-runtime::partial$mmp-concept.adoc[tags=!authenticationMethodsIntro;!authenticationCloudHub;!authenticationRTF]
+=======
+include::mule-runtime::partial$mmp-concept.adoc[tags=authenticationOptionsList;!authenticationCloudHub;!authenticationRTF]
+>>>>>>> v4.3
```

Comment:
Which of these is correct to keep in v4.4? 
* from v4.4: tags=!authenticationMethodsIntro
* from v4.3: tags=authenticationOptionsList

Reply:
@Cristian-Venticinque

Reply:
Line 131 is correct, this was to fix an incorrect include directive in some MMP articles.